OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -51,7 +51,7 @@ public FindResources(Actor self, CPos avoidCell)
 
 		public override Activity Tick(Actor self)
 		{
-			if (IsCanceled || NextActivity != null)
+			if (IsCanceled || NextInQueue != null)
 				return NextActivity;
```

Comment:
Splitting this into two checks make this clearer?  Code that checks if something isn't null and then returns something else is usually a copy-paste error, so for my future sanity it would be nice if we could change this or have a comment `// Yes, I really do mean to check NextInQueue but return NextActivity`